### ISSUE TITLE
"Increase search radius" button on select city

### ISSUE THREAD

**Issue by OpenSourceForEver** [osmandapp/Osmand#3838]
Latest osmand nightly build.  Offline map Spain May 1, 2017
Search>  adress>  select city. 
What is the sense of "increase search radius"? If I want search a far town this button is not helpfull.

**Comment by Gizmot** [comment]
I do agree, since osmand went "Google text bar" for searching for an address has turned from bad to useless. Even if you go search by city, its always searching by radius, which is fine if you never go out of town. You NEED to be able to enter what country, state/province, then town.

**Comment by vshcherb** [comment]
For select city context it increases radius much bigger comparing to "contextless" search. It should reach 3000 km at least after first hit.

**Comment by OpenSourceForEver** [comment]
Without pressing "Increase search radius"
![screenshot_2017-06-02-01-25-51](https://cloud.githubusercontent.com/assets/12786225/26704984/4a3c56f6-4733-11e7-8c1b-c4b581013915.png)

**Comment by OpenSourceForEver** [comment]
After pressing "Increase search radius" it appears a village 330 km from me. The same village do not appear before pressing "Increase search radius".

Sorry but I do not know the reasson but I can not upload screenshoots.

**Comment by vshcherb** [comment]
I received the screenshot, yes it is a known issue today. The search of villages is limited by radius it is a trade-off solution for other search queries.

**Comment by OpenSourceForEver** [comment]
I think this is fixed because https://github.com/osmandapp/Osmand/issues/6126 is fixed.
What do you think @Gizmot ?

**Comment by Gizmot** [comment]
My point of view is based on my personnal use case: international business traveller.
Lets say I`m trying to save as a favorite a hotel location in Belgium for an upcoming trip while in North America. The work around (that you need to figure out first) and he only way to hope finding it is to know exactly where it is on a world map then zoom all the way in. It's not really a search. 
Now, if you increase search search radius everytime by 1000km, I will need to press about 12 times. I havent tried it, but I am not sure how OSM will handle searching half the globe surface for a road name in Dutch.

**Comment by OpenSourceForEver** [comment]
Have you downloaded the lastest nightly build to test it?

**Comment by Gizmot** [comment]
no,  I have a brand new baby girl so really no time for that right now
